tutorials/setup_tutorial.md

```diff
@@ -0,0 +1,255 @@
+# Tutorial: Getting Started 
+
+
+Wecome to `jiant`! Let's help get you set up and running a demo experiment!
+
+## 1. Install
```

Comment:
You skip from 1 to 3. Just delete the numbers?

Reply:
I do have 2 below

Reply:
Oops—sorry. Indentation threw me off. (But won't be rendered, I assume...)